public/docs/_examples/quickstart/ts/package.1.json

```diff
@@ -13,7 +13,6 @@
   "dependencies": {
     "@angular/common": "2.0.0-rc.6",
     "@angular/compiler": "2.0.0-rc.6",
-    "@angular/compiler-cli": "0.6.0",
```

Comment:
Sure, why not. Doesn't hurt the readers, only us, but doesn't give us anything by having it there right now.